src/riak_kv_qry_coverage_plan.erl

```diff
@@ -73,8 +73,13 @@ do_plan({Query, Bucket}, NVal, _PVC, ReqId, NodeCheckService, _Request, DownNode
     UnavailableVNodes =
         riak_core_coverage_plan:identify_unavailable_vnodes(CHBin, RingSize,
                                                             NodeCheckService),
-
-    Key = make_key(Query),
+    %% access the select record using element so that we don't have to use the
+    %% record syntax, the record may have been upgraded or downgraded at this
+    %% point
+    HelperMod = element(?SQL_SELECT.helper_mod, Query),
```

Comment:
I wonder if these should be accessor functions? Then they could be reused.  Just a suggestion.